.golangci.yml

```diff
@@ -0,0 +1,35 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 3m
+
+linters:
+  enable:
+    - deadcode
+    - errcheck
+    - gas
+    - goconst
+    - gofmt
+    - goimports
+    - golint
+    - gosimple
+    - ineffassign
+    - misspell
+    - nakedret
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+  disable:
```

Comment:
Maybe it’s better to set disable-all:true? It will do the same but will allow easier updates of golangci-lint when new linters will be added

Reply:
I've decided to mention all the disabled checkers. Just for a clarity :)